### ISSUE TITLE
${message.subject} does not output the actual email subject

### ISSUE THREAD

**Issue by jiho** [robertklep/quotefixformac#97]
When using ${message.subject} in a custom Forward attribution (on Catalina, Mail Version 13.4 (3608.120.23.2.4)), the actual subject is garbled in the forwarded email.

```
On 2021-01-06 at 10:13, A*** M*** < a***.m***@yahoo.com> sent:
Subject: 1cfc
To: J*** I***
```

When the subject should have been "Nouvelle année".

PS: names and emails obfuscated of course.

**Comment by robertklep** [comment]
Yes, I believe this is a known issue with Catalina (and probably Big Sur). I'm still running Mojave so can't debug or fix :(

**Comment by jiho** [comment]
OK. Thanks for the fast reply!
Happy to help in any way I can. I have XCode command line tools installed but not the full developer tools. I do a fair bit of Python and other similar stuff but know next to nothing about ObjC.
I could run a version with debugging info activated if you provide me with one and send you a trace/log.

**Comment by robertklep** [comment]
@jiho to be honest, I have no idea where to start looking 🥴 the value looks like a pointer value of some sorts.
